cns/restserver/ipam.go

```diff
@@ -412,7 +536,12 @@ func (service *HTTPRestService) assignIPConfig(ipconfig cns.IPConfigurationStatu
 		return err
 	}
 
-	service.PodIPIDByPodInterfaceKey[podInfo.Key()] = ipconfig.ID
+	if service.PodIPIDByPodInterfaceKey[podInfo.Key()] == nil {
```

Comment:
can you add a todo in func (service *HTTPRestService) updateIPConfigState to update only if the current state is different than new state?